app.py

```diff
@@ -313,6 +304,9 @@ def _request_end(resp):
 
 tuf_metadata_api = TUFMetadataAPI(app, app.config)
 
+marketplace_users = MarketplaceUserApi(app)
+marketplace_subscriptions = MarketplaceSubscriptionApi(app)
```

Comment:
Are we deprecating `ENTITLEMENT_RECONCILIATION`?

Reply:
No, the flag is still used for enabling the reconciler worker. This change was mostly for allowing the mocked user and subscription apis to be re-used in unit tests.